addons/account/static/src/components/bills_upload/bills_upload.js

```diff
@@ -134,7 +134,7 @@ export class AccountMoveUploadListRenderer extends ListRenderer {
 
     setup() {
         super.setup();
-        this.state.dropzoneVisible = false;
+        this.state = useState({ dropzoneVisible: false });
```

Comment:
Maybe not a good patter to define this.state when you extends a View Component.
Another addon, could patch ListRenderer and add this.state.